src/components/button/icons/user-icon.jsx

```diff
@@ -0,0 +1,16 @@
+const UserIcon = () => (
```

Comment:
This is not the correct way to use svg. Either use it directly in code where required or ideally create an svg file in assets folder and use that.

Reply:
ok using now directly. thanks